design.md

```diff
@@ -2,12 +2,24 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+Lord of the Rings fans who are moderately well-versed in the lore of the world.
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+Because it is fun (we are LotR fans) and because it also provides essentially equivalent functionality without major changes to the program model.
+
 ## What behaviors are easier to express in your design than in Picobot’s original design?  If there are no such behaviors, why not?
 
+None.  We essentially took the existing syntax and made it more verbose.  Yay!
+
 ## What behaviors are more difficult to express in your design than in Picobot’s original design? If there are no such behaviors, why not?
 
+None of them are more difficult conceptually, they simply require more verbosity (which may be easier for some people, because it is more clear what is happening).
+
 ## On a scale of 1–10 (where 10 is “very different”), how different is your syntax from PicoBot’s original design?
 
+4. It uses different words but equivalent grammar to express the same concepts.
+
```

Comment:
After reviewing I think I'd say it was more like a 6 or 7. It asks your different your syntax is and it is almost completely different despite the fact that your underlying model is very similar